testscommon/enableEpochsHandlerStub.go

```diff
@@ -981,6 +982,14 @@ func (stub *EnableEpochsHandlerStub) IsRuntimeMemStoreLimitEnabled() bool {
 	return stub.IsRuntimeMemStoreLimitEnabledField
 }
 
+// IsRuntimeCodeSizeFixEnabled -
+func (stub *EnableEpochsHandlerStub) IsRuntimeCodeSizeFixEnabled() bool {
```

Comment:
a lot of checks failed because the other epochs handler stubs need to be changed also

Reply:
Fixed all.